src/components/FiveCardLayout.js

```diff
@@ -0,0 +1,39 @@
+import React, {Component, PropTypes} from 'react'
+import Card from './Card'
+import CardBack from './CardBack'
+import _ from 'lodash'
+
+export default class FiveCardLayout extends Component {
+
+  static propTypes = {
+    cards: PropTypes.array.isRequired,
+
+  }
+
+  renderCards() {
+    // assign positions
```

Comment:
Like the comment in the beginning of the review, are comments necessary for this?